.semaphore/semaphore.yml

```diff
@@ -33,6 +33,7 @@ blocks:
 
         - name: codenarc analysis
           commands:
+            - rm -rf $HOME/.m2
```

Comment:
I am not sure of cache implementation in semaphore, but this will probably affect other jobs. Plus, we are still wasting time in container to restore cache, just to delete it. It is better to create new "no cache" build like so and add codenarc job to it:

```yml
  - name: "Linux openjdk build (no cache)"
    task:
      prologue:
        commands:
          - checkout
          - sudo apt-get update
          - sudo apt-get install -y ant groovy xsltproc xmlstarlet
          - export GRADLE_OPTS=-Dorg.gradle.console=plain

```

Reply:
It is few seconds, now we wasting more on mentors side to recheck CI status.

Semaphore has parallelization of 5 task, they must run them in separate containers.

We can keep issue open to do bigger update as you suggested.
I just tried to render some fix in 10 minutes that I got. And ease pain to recheck CI status.

Reply:
Ok, I added comment to issue, first priority is stable CI.